pkg/specgen/generate/storage.go

```diff
@@ -20,9 +20,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var (
-	errDuplicateDest = errors.Errorf("duplicate mount destination")
-)
+var errDuplicateDest = errors.Errorf("duplicate mount destination")
+
+const containerInitPath = "/sbin/podman-init"
```

Comment:
nit: Should this be placed somewhere here https://github.com/containers/common/blob/main/pkg/config/default.go#L48 however this does not seems to be configurable so not a blocker.

Reply:
At the moment, I don't think this is something a user should worry or care about.